server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -270,12 +270,12 @@ class portaudio_backend : public detail::audio_backend_base<sample_type, float,
         if (statusFlags & (paInputOverflow | paInputUnderflow | paOutputOverflow | paOutputUnderflow))
             engine_functor::sync_clock();
 
-        const float* inputs[input_channels];
+        const float** inputs = new const float*[input_channels];
```

Comment:
this is the rt-audio thread, so `new` is forbidden. msvc doesn't support VLA, but it supports `_alloca` (also, if you `new`, without `delete`, you'll leak memory

Reply:
thanks!